components/resource_control/src/resource_group.rs

```diff
@@ -477,7 +477,7 @@ impl ResourceController {
 
         // TODO: use different threshold for different resource type
         // needn't do update if the virtual different is less than 100ms/100KB.
-        if min_vt + 100_000 >= max_vt && max_vt < RESET_VT_THRESHOLD {
+        if min_vt.wrapping_add(100_000) >= max_vt && max_vt < RESET_VT_THRESHOLD {
```

Comment:
FYI, compiler thinks here maybe cause `attempt to add with overflow`

Reply:
if min_vt >= max_vt.saturating_sub(1000)